src/content/guidelines/color/overview.md

```diff
@@ -0,0 +1,185 @@
+title: Color
+tabs: ['Overview', 'Usage']
+---
+
+<anchor-links>
+<ul>
+    <li><a href="#resources">Resources</a></li>
+    <li><a href="#introduction">Introduction</a></li>
+    <li><a href="#ibm-color-anatomy">IBM Color anatomy</a></li>
+    <li><a href="#tokens">Tokens</a></li>
+    <li><a href="#themes">Themes</a></li>
+    <li><a href="#interaction-states">Interaction states</a></li>
+    <li><a href="#digital-accessibility">Digital accessibility</a></li>
+</ul>
+</anchor-links>
+
+## Resources
+
+<grid-wrapper col_lg="8" flex="true" bleed="true">
+<clickable-tile
+    title="Color palettes for Adobe™(.ase)"
+    href="https://w3.ibm.com/design/language//IBM-color-palettes.zip"
+    type="resource"
+    >
+    <img src="images/ase.png" alt="Color palettes for Adobe" />
+</clickable-tile>
+</grid-wrapper>
+
+## Introduction
+
+Carbon uses _tokens_ and _themes_ to manage color. Tokens are role-based, and themes specify the color values that serve those roles in the UI. 
+
+### Color terms
+
+| Term      | Definition                                                                                                  |
+| --------- | ----------------------------------------------------------------------------------------------------------- |
+| Theme | The set of unique values assigned to the tokens of a Carbon interface                                                            |
+| Token | The code identifier for a unique role or set of roles. Tokens are universal and never change across themes. |
+| Role | The systematic usage(s) of a token. Roles cannot be changed between themes.                                  |
+| Value | The actual style (i.e. hex code) assigned to a token                         |
+
+
+## IBM color anatomy
+
+Carbon's default themes are derived from the IBM Design Language color palette. The Neutral Gray family is dominant in the default themes, making use of subtle shifts in value to help organize content into distinct zones. 
+
+The core Blue family serves as the primary action color across all IBM products and experiences. Additional colors are used sparingly and purposefully.
+
+![Color anatomy](images/color_anatomy_bkg.svg)
+
+## Tokens
+
+Color in Carbon is managed using color tokens. Tokens are a method of abstracting color by role or usage, independent of the actual color values desired for any specific case. For example, instead of needing to code all instances of input labels in a UI to be `#565656`, the _form_ component in Carbon specifies the token `$text-02` for these lables, and that token is specified in a separate theme file to be `#565656`. For a different theme, that same `text-02` token could be mapped to a different hex value, like `#ffffff`. Thus, tokens not only allow for more efficient color updates within a theme, but also enable any UI (or portion of a UI) built with Carbon to easily switch between different themes. 
```

Comment:
"Tokens are a method of abstracting color by role or usage, independent of the actual color values ~desired for any specific case~."
Sounds clearer without the last but but just my opinion